docs/docs-beta/docs/guides/ingesting-data.md

```diff
@@ -2,54 +2,56 @@
+- **Coordinate data ingestion with downstream data transformation,** such as rebuilding a set of dbt models after upstream data is updated
 - **Represent ingested data assets in an asset graph**, which enables understanding what ingested data exists, how ingested data is used, and where data is ingested from
 
 ## Orchestrating data ingestion tools
 
-Dagster integrates with a variety of data ingestion tools, enabling you to sync diverse data sources into your data warehouse tables using pre-built connectors. With these integrations, Dagster allows you to:
+Dagster integrates with a number of data ingestion tools, enabling you to sync diverse data sources into data warehouse tables using pre-built connectors. With these integrations, Dagster allows you to:
 
 - Represent ingested tables as assets within the Dagster asset graph
 - Trigger asset materializations that automatically invoke these tools to initiate data syncs
 
 Dagster currently integrates with the following data ingestion tools:
 
-- [Airbyte](/todo)
-- [dlt](/todo)
-- [Fivetran](/todo)
-- [Sling](/todo)
+- Airbyte
+- dlt
+- Fivetran
+- Sling
 
 ## Writing custom data ingestion pipelines
 
-Writing code in a language like Python to ingest data into a platform is also a common approach. This is especially useful when you have unique data ingestion requirements that aren't addressed by existing tools, or when you prefer to keep your platform streamlined without adding new tools.
+Using a language like Python to write code for data ingestion into a platform is also a common approach. This is useful when you have unique data ingestion requirements that aren't addressed by existing tools, or when you prefer to keep your platform streamlined without adding new tools.
 
-For example, imagine there's a CSV file of counties on the internet and you want to load it into your Snowflake data warehouse as a table. To do this, you might directly define an asset that represents that table in your warehouse. The asset's materialization function fetches data from the internet and loads it into that table:
+For example, imagine there's a CSV file of counties on the internet and you want to load it into your Snowflake data warehouse as a table. To do this, you might define a Dagster asset that represents that table in your warehouse. The asset's materialization function fetches data from the internet and loads it into that table:
 
-<CodeExample filePath="guides/data-ingestion/custom-data-ingestion.py" language="python" title="Custom data ingestion" />
+<CodeExample filePath="guides/data-ingestion/custom-data-ingestion.py" language="python" />
 
 ## Next steps
 
-{/* TODO add next steps */}
-
-- Learn how to [transform data using Dagster's dbt integration](/guides/transform-dbt)
+- Transform data using [Dagster's dbt integration](/guides/transform-dbt)
+- Run non-Python code with [Dagster Pipes](/guides/non-python)
```

Comment:
I'm not sure if Pipes is a relevant follow-up, but data quality / asset checks might be!